content/rs/concepts/shard-placement-policy.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: Shard Placement Policy
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+aliases: /rs/concepts/rebalancing-shard-placement/
+---
+{{< embed-md "shard-placement-intro.md"  >}}
+
+## Shard Placement Policy
+
+The shard placement policies are:
```

Comment:
Redis Enterprise supports two shard placement policies:

Reply:
I don't think this helps the reader so much and I prefer to avoid giving a number to avoid mistakes if the number changes. These mistakes happen a lot.